src/mixed-line-bar-chart/utils.ts

```diff
@@ -108,6 +126,8 @@ export function calculateOffsetMaps(
 
     return acc;
   }, [] as StackedOffsets[]);
+
+  return { minValues, maxValues, seriesOffsets };
```

Comment:
If `seriesOffsets` already contains sorted data, isn't it self explanatory which item is min and which item is max? Do you really need a separate value here?

Reply:
The `seriesOffsets` includes data for a single series only. For example:

```
series = "John"
seriesOffsets = {
  positive: { "Apples": 0, "Oranges": 5 },
  negative: { "Pears": 0  }
}
```

The chart is rendered series by series, so the John series will render right above the X axis for "Apples", above the X axis and some other series for "Oranges", and right below the X axis for "Pears". This carries no information about other series.

Reply:
Thanks for the comment though, looking at it now I realised it worth taking a step back and create a better data structure than series offsets to accommodate all needs.